R/corrpart.b.R

```diff
@@ -246,72 +233,59 @@ corrPartClass <- R6::R6Class(
 
             return(data)
         },
-        .test = function(var1, var2, controls, type) {
+        .test = function(var1, var2, controls, type, hyp) {
 
             data <- private$.cleanData(var1, var2)
             var1 <- data[[var1]]
             var2 <- data[[var2]]
 
-            if (type != 'zero' && length(controls) > 0)
-                controls <- data[, controls]
-
-            results <- list()
+            n <- length(data[[1]])
+            d <- 1 + as.integer(type == 'semi')
+            c <- length(controls)
+            if (c == 0) { rdta <- cbind(var1, var2) } else { rdta = cbind(var1, var2, data[, controls]) }
```

Comment:
Same as before, no inline if statements preferably. Also, you're using an arrow assignment in one block, and equal-sign assignment in the other. Bot should be arrow assignments. And what does `rdta` stand for?

Reply:
rdta (raw data) - we could continue with using data but I did not want to overwrite it.